packages/hub/services/data-integrity-checks/scheduled-payments.ts

```diff
@@ -10,6 +14,15 @@ export const UNATTEMPTED_ALLOWED_MINUTES = 24 * 60;
 export const ATTEMPTS_WITHOUT_TX_HASH_ALLOWED_MINUTES = 60;
 export const UNFINISHED_ATTEMPT_DURATION_ALLOWED_MINUTES = 24 * 60;
 
+// Relayer needs funds to pay for safe transactions gas (e.g creating safes, scheduling a payment).
```

Comment:
Note to self: remove "creating safes" – that actually comes from the user's wallet!